File: insert_at_end.c
// Program: Insert at End of Array
// Author: Albin Mathew
#include <stdio.h> 

int main() { 
    int arr[100]; // Array to store numbers, max size 100
    int n; // Variable to store how many elements in array
    int element; // Variable to store the new element we want to add
    int i; // Loop variable for printing array

    // Ask user how many elements in array
    printf("Enter number of elements in array: ");
    scanf("%d", &n); // Store user input in n

    // Check if array is not full (less than 100)
    if (n >= 100) {
        printf("Array is full, cannot add more elements!\n");
        return 1; // Exit program if array is full
    }

    // Ask user to input array elements
    printf("Enter %d elements:\n", n);
    for (i = 0; i < n; i++) { // Loop to take input for each element
        scanf("%d", &arr[i]); // Store each element in array
    }

    // Ask user for the element to add at end
    printf("Enter the element to insert at end: ");
    scanf("%d", &element); // Store the new element

    // Add element at the end of array
    arr[n] = element; // Put new element at index n (end of array)
    n++; // Increase size of array by 1

    // Print the updated array
    printf("Array after inserting %d at end:\n", element);
    for (i = 0; i < n; i++) { // Loop to print all elements
        printf("%d ", arr[i]); // Print each element with space
    }
    printf("\n"); 

    return 0; 
}

File: insert_at_end.py
// Program: Insert at End of Array
// Author: Albin Mathew
# Create an empty list to store numbers
arr = []

# Ask user how many elements they want in list
n = int(input("Enter number of elements in array: "))

# Ask user to input the elements
print("Enter", n, "elements:")
for i in range(n): # Loop to take n elements
    element = int(input()) # Take each element as input
    arr.append(element) # Add element to the end of list

# Ask user for the element to add at end
new_element = int(input("Enter the element to insert at end: "))

# Add the new element at the end of list
arr.append(new_element) # Append adds element at the end

# Print the updated list
print("Array after inserting", new_element, "at end:")
for i in range(len(arr)): # Loop to print all elements
    print(arr[i], end=" ") # Print each element with space
print() # New line after printing list